app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -185,9 +185,9 @@
   private ViewGroup.MarginLayoutParams audioBarParams;
   private boolean isInMultiWindowMode;
 
-  private String[] translationItems;
-  private List<LocalTranslation> translations;
-  private Set<String> activeTranslations;
+  private String[] translationSpinnerTitles;
```

Comment:
how about `translationNames` instead?

Reply:
it was hard to me to understand the code since there is 3 or 4 variables related to `translations` so I renamed this variables , but it's OK to revert it back. 

I think it will better if we add `isActive` column to `translations' table, it will be more concise and readable.

Reply:
we can also start migrating gradually  to `Room` ,`Hilt` , `livedata`, `MVVM` , is there in plan for that , where can we start ?

Reply:
This project started over 10 years ago, which is why there's so much legacy code here - in sha' Allah planning to modernize, just need to allocate some more time to it.

* Hilt - we're already using Dagger, and I prefer Dagger + Anvil from Hilt from usage in other projects. 

* Room - either it or SQLDelight, there's an issue somewhere on the tracker about it. Not sure which yet.

* LiveData - prefer not to use it - we should be moving more towards unidirectional flows though, which most of the new code is doing.

* MVVM - doesn't really matter VM vs P as long as we are respecting things (unidirectional flows, streams for communications, etc) and keeping the business logic layer testable.

Reply:
I think we can start with converting all code to Kotlin

Reply:
how about coroutine ?

Reply:
re: converting code to Kotlin - at 28.6% Kotlin now, just converting things when i am going to work in those classes, so definitely an on-going process. @wiryadev has been opening some PRs also recently.

re: coroutines - the more recently written code from this year is already using coroutines - plan is to deprecate Rx moving forward.